app/pages/project/instances/instance/tabs/StorageTab.tsx

```diff
@@ -60,7 +61,11 @@ StorageTab.loader = async ({ params }: LoaderFunctionArgs) => {
   return null
 }
 
-const colHelper = createColumnHelper<Disk>()
+type InstanceDisk = Disk & {
+  instanceState: InstanceState
+}
```

Comment:
By making the instance state part of the row data, we can avoid having to remake the row actions callback whenever the instance state changes.